src/bindings/python/tests/test_imported_entity.py

```diff
@@ -19,13 +19,15 @@ def test_named_entity(self):
         # Test access to inherited methods
         x = ImportedEntity()
         idx = 'test'
-        self.assertEqual(x.getId(), '')
+        self.assertIs(x.getId(), '')
```

Comment:
Should really keep the use of `assertEqual` for any non-identity tests (i.e. test for `None`, or that the value assigned to one name is also to another name).

Reply:
Could you explain a bit more? I don't understand what you're saying here!

Reply:
`assertIs` is used to assert the exact same object (e.g. two assignments (variables) that point to the same object), and `assertEqual` asserts actual equality which is what is usually wanted.  See [this](https://stackoverflow.com/questions/7281774/whats-the-difference-between-assertequal-and-assertis-assertis-was-introduced) and [this](https://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce?noredirect=1&lq=1) stackoverflow posts.

Reply:
Thanks! I see what you mean now.

I think I was a bit worried that a `False` or a `[]` would get through, as in `if not '': print('hi')` and `if not False: print('hi')`, but on closer inspection those semantics don't apply here.

Will change!

Reply:
Done!